specutils/io/default_loaders/jwst_reader.py

```diff
@@ -155,18 +233,25 @@ def _jwst_x1d_loader(file_obj, **kwargs):
             # SRCTYPE used to be in primary header, but was moved around. As
             # per most recent pipeline definition, it should be in the
             # EXTRACT1D extension.
+            #
+            # SRCTYPE should either be POINT or EXTENDED.  In some cases, it is UNKNOWN
+            # or missing.  If that's the case, default to using POINT as the SRCTYPE.
+            # Error out only when SRCTYPE is a bad value.
             srctype = None
             if "srctype" in hdu.header:
-                srctype = hdu.header.get("srctype")
+                srctype = hdu.header.get("srctype", None)
+
+            # checking if SRCTPYE is missing or UNKNOWN
+            if not srctype or srctype == 'UNKNOWN':
+                log.warning('SRCTYPE is missing or UNKNOWN.  Defaulting to srctype="POINT".')
```

Comment:
I find logging messages like this are much easier to understand if they say where they come from (of course one can search the line number, but most people don't...)
```suggestion
                log.warning('SRCTYPE is missing or UNKNOWN in JWST x1d loader. 
                             Defaulting to srctype="POINT".')
```